### PR TITLE
Add Bugsnag to Frontend

### PR DESCRIPTION
**Pull request by RossIV**

### CODE REVIEW

File: resources/js/components/Kiosk.vue [hunk 1]
```diff
@@ -268,25 +287,8 @@ export default {
...
-                        });
-                    } else {
-                        this.$swal.fire({
-                            title: 'Error',
-                            text: 'Unable to process data. Check your internet connection or try refreshing the page.',
-                            icon: 'error',
-                        });
-                    }
+                    this.axiosErrorToBugsnag(error)
+                    this.axiosErrorToBugsnag(error)
```
Comment (kberzinch) [inline resources/js/components/Kiosk.vue:291]: Is this intentionally duplicated?

Comment (JoshOldenburg) [reply resources/js/components/Kiosk.vue:291]: I think this should be `handleAxiosError`.

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]
Looks good once the duplicated call is fixed.

File: resources/js/components/Kiosk.vue (hunk 1 above)
Comment (RossIV) [reply resources/js/components/Kiosk.vue:291]: No

**Review by kberzinch — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**